src/components/Asset/AssetActions/index.tsx

```diff
@@ -81,6 +81,18 @@ export default function AssetActions({
             )
           : await getFileDidInfo(asset?.id, asset?.services[0]?.id, providerUrl)
         fileInfoResponse && setFileMetadata(fileInfoResponse[0])
+
+        // set the content type in the Dataset Schema
+        const datasetSchema = document.getElementById('DatasetSchema')
```

Comment:
Direct DOM manipulation is quite adventurous in a React app. For changing anything in the `head` of any page, we'll have to use [`next/head`](https://nextjs.org/docs/api-reference/next/head). Next.js devs also suggest to use [`next/script`](https://nextjs.org/docs/basic-features/script) when adding `script` to `head` but we might not use this in this case as we are not loading external scripts with it.